test/validation/api/ipsec/ipsec.c

```diff
@@ -16,11 +16,25 @@
 #include "test_vectors.h"
 #include "reass_test_vectors.h"
 
+#define MAX_EVENT_CACHE 3
+
+struct sched_event_cache_s {
```

Comment:
The type name could be cached_event_s or something like that since the type represents one cached event instead of the whole cache and the subsequent array is an array of cached events, not an array of event caches.

Reply:
Ack. Renamed to cached_event_s.